doc/apiary/v2/fiware-ngsiv2-rc-2016_05_31.apib

```diff
@@ -32,16 +32,20 @@ Sergio García Gómez (Telefónica I+D), Martin Bauer (NEC).
   
 ## Status
 
-This is a work in progress and is changing on a daily basis.
-Please send your comments to fiware-ngsi@lists.fiware.org.
-You can trace the discussions checking the archives of the mailing list:
-https://lists.fiware.org/private/fiware-ngsi/ (list subscription required).
+This specification is a release candidate of the NGSIv2 API specification. By "release candidate"
+we mean that the specification is quite stable, but changes may occur with regard to new release
```

Comment:
I would rephrase this as "Only Minor Changes are expected to the API described by this document". Nonetheless new functionalities will be added, particularly JSON-LD and NGSI9 (context registration) support.